charts/kargo/templates/controller/cluster-roles.yaml

```diff
@@ -114,7 +114,6 @@ rules:
   - create
   - get
   - list
-  - patch
```

Comment:
I don't believe there is actually any need to patch AnalysisRuns. Everything works correctly e2e without this.

@hiddeco maybe you can give me a sanity check on this one, however.

Reply:
nm. I just noticed the DM you sent me about this hours ago.

It's for aborting a verification's underlying AnalysisRun.

I'll fix this in the morning.